astropy/io/ascii/latex.py

```diff
@@ -1,9 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """An extensible ASCII table reader and writer.
-
```

Comment:
Please revert all these line changes.  I think your editor was configured (maybe?) to remove blank lines.  But many of these blank lines are actually necessary for the docstrings to parsed and displayed correctly.